others/sparse_matrix_with_vectors.cpp

```diff
@@ -0,0 +1,77 @@
+/*
+This program checks if a given matrix is sparse or not
+i.e in a spaarse matrix zero valued elements are much more as compared to no zero values elements
+here if zero valued elemts are more than the 50% size of the matrix, we will call it a sparse matrix
+*/
+#include<iostream>
+#include<vector>
+#include<cassert>
+
+using namespace std;
+/*
+counts the number of  zero valued elements
+check if the count is more than the half of the size of matrix
+
+*/
+bool is_sparse_matrix(vector<vector<int>> vect)
+{
+    int zero_counter = 0;
+
+     for(int i =0; i<vect.size(); i++)
+    {
+        for(int j = 0; j<vect[i].size(); j++)
+        {
+            if (vect[i][j]==0)
+            {
+                zero_counter+=1;
+            }
+        }
+    }
+
+    if(zero_counter>((vect.size()* vect[0].size())/2))
+    {
+        return 1;
+    }
+    else
+    {
+        return 0;
+    }
```

Comment:
```suggestion
   return !(zero_counter > (vect.size() * vect[0].size()) * 0.5
```
this could just return the result of the condition